### PR TITLE
[WIP][Training - My] - Training research advanced ruby

### PR DESCRIPTION
**Pull request by truong-my-mulodo**
## Task list

1.  Migration

    - [x] Create/Crop database
    - [x] Create/Edit/Delete tables
    - [x]  Add/Edit/Delete columns
    - [x] Generate data by seeds
    - [x] Generate migration
    - [x] Rollback migration

2. Create, update, delete Article

### CODE REVIEW

File: app/views/articles/_form.html.slim [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  div.form-group
+    = form.label t('activerecord.attributes.article.title')
+    = form.text_field :title, id: :article_title, class: 'form-control'
+
+  div.form-group
+    = form.label t('activerecord.attributes.article.description')
+    = form.text_area :description, id: :article_description, class: 'form-control'
+
+  / div.form-group
```
Comment (tran-hai-mulodo) [inline app/views/articles/_form.html.slim]: @truong-my-mulodo 

Please remove unuse code here.

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -34,7 +34,8 @@ html
           = link_to " Sign in ", new_user_session_path
       p  
         'Select Date:
-        = datetime_select( "article", "written_on", start_year: 1995)
+        / = datetime_select( "article", "written_on", start_year: 1995)
```
Comment (tran-hai-mulodo) [inline app/views/layouts/application.html.slim]: @truong-my-mulodo

Please remove unuse code here.

File: config/locales/en.yml [hunk 3]
```diff
@@ -72,6 +72,11 @@ en:
         content: "Content"
         quantity: "Quantity"
         image: "Image"
+      article:
```
Comment (tran-hai-mulodo) [inline config/locales/en.yml]: @truong-my-mulodo

We need to add some other languages such as: 'es', 'ja',... in order to show the multi language support.

**Comment by truong-my-mulodo** [comment]
I fixed them, please check it again !
Thanks

**Comment by nguyen-dat-mulodo** [comment]
I reviewed it. it's OK, thanks.

**Comment by tran-hai-mulodo** [comment]
@truong-my-mulodo 
cc: @nguyen-dat-mulodo 

We'll use "training-truong-my" as the base branch for the training course. So, I'd like to close this PR. Thanks for your hard work.

**Pull request closed without merging**